homeassistant/components/template/alarm_control_panel.py

```diff
@@ -88,7 +101,8 @@ async def _async_create_entities(hass, config):
         arm_away_action = device_config.get(CONF_ARM_AWAY_ACTION)
         arm_home_action = device_config.get(CONF_ARM_HOME_ACTION)
         arm_night_action = device_config.get(CONF_ARM_NIGHT_ACTION)
-        code_arm_required = device_config[CONF_CODE_ARM_REQUIRED]
+        code_arm_required = device_config.get(CONF_CODE_ARM_REQUIRED)
+        code_format = device_config.get(CONF_CODE_FORMAT)
```

Comment:
Both of these have default values, don't use `dict.get`.
```suggestion
        code_arm_required = device_config[CONF_CODE_ARM_REQUIRED]
        code_format = device_config[CONF_CODE_FORMAT]
```